### PR TITLE
Small documentation updates

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/systemd.resource-control.xml [hunk 1]
```diff
@@ -918,31 +918,34 @@ DeviceAllow=/dev/loop-control
...
+          adversely affect <command>systemd-oomd</command>'s kill behavior. Also note that these extended
+          attributes are not applied recursively to cgroups under this unit's cgroup.</para>
+
+          <para>Defaults to <option>none</option> which means no extended attributes will be set and
+          systemd-oomd will sort this unit's cgroup as defined in
           <citerefentry><refentrytitle>systemd-oomd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-          and <citerefentry><refentrytitle>oomd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry> (if this
-          unit's cgroup becomes a candidate).</para>
+          and <citerefentry><refentrytitle>oomd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+          (if this unit's cgroup becomes a candidate).</para>
```
Comment (poettering) [inline man/systemd.resource-control.xml]: quite frnakly i'd drop much of this from the docs. It documents implementation details, which then might become API, and they shouldn't be API

File: man/timesyncd.conf.xml [hunk 2]
```diff
@@ -47,54 +47,59 @@
...
-        <varname>PollIntervalMinSec=</varname> defaults to 32 seconds, and
-        <varname>PollIntervalMaxSec=</varname> defaults to 2048 seconds.</para></listitem>
+        <listitem><para>The minimum and maximum poll intervals for NTP messages. Polling starts at the
+        minimum poll interval, and is adjusted within the specified limits in response to received packets.
+        </para>
+
+        <para>Each setting takes a time value. The default unit is seconds, but other units may be specified,
+        see <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
+        <varname>PollIntervalMinSec=</varname> defaults to 32 seconds and must not be smaller than 16
+        seconds. <varname>PollIntervalMaxSec=</varname> defaults to 2048 seconds and must be larger than
```
Comment (poettering) [inline man/timesyncd.conf.xml]: 2048s is a weird thing to say. Maybe say `34min 8s`

File: man/hostnamectl.xml [hunk 3]
```diff
@@ -76,9 +79,13 @@
         still following the validity rules of the specific name. This simplification of the hostname string is not done
         if only the transient and/or static hostnames are set, and the pretty hostname is left untouched.</para>
 
-        <para>Pass the empty string <literal></literal> as the
-        hostname to reset the selected hostnames to their default
-        (usually <literal>localhost</literal>).</para></listitem>
+        <para>The static and transient hostnames must each be either a single DNS label (a string composed of
+        7-bit ASCII lower-case characters and no spaces or dots, limited to the format allowed for DNS domain
+        name labels), or a sequence of such labels separated by single dots that forms a valid DNS FQDN. The
+        length must be at most 64 characters.</para>
```
Comment (poettering) [inline man/hostnamectl.xml]: might be nice to say that this size limitation is not a DNS limitation, but a Linux one.

Maybe: "Note that unlike for DNS FQDNs a size limit of 64 characters is enforced."

File: man/systemd.resource-control.xml [hunk 4]
```diff
@@ -63,6 +63,25 @@
     url="https://www.freedesktop.org/wiki/Software/systemd/ControlGroupInterface/">New
     Control Group Interfaces</ulink> for an introduction on how to make
     use of resource control APIs from programs.</para>
+
+    <refsect2>
+      <title>Setting configuration for a group of related units</title>
```
Comment (poettering) [inline man/systemd.resource-control.xml]: configuration → Resource Controls?

**Comment by keszybz** [comment]
Updated with the suggested changed.

**Pull request merged**